crates/fuel-gas-price-algorithm/src/v0.rs

```diff
@@ -85,15 +98,14 @@ impl AlgorithmUpdaterV0 {
             .unwrap_or(self.l2_block_fullness_threshold_percent);
 
         match fullness_percent.cmp(&self.l2_block_fullness_threshold_percent) {
-            std::cmp::Ordering::Greater => {
+            std::cmp::Ordering::Greater | std::cmp::Ordering::Equal => {
```

Comment:
This is arbitrary. It made the tests easier to write and we don't have any specs on how the threshold should behave.